where-to-travel/src/main/webapp/script.js

```diff
@@ -332,11 +332,11 @@ function populatePlaces(placeArray) {
 
   $('.icon').click(function() {
     $(this).toggleClass('press');
-    if (firebase.auth().currentUser ) {
+    if (firebase.auth().currentUser && $(this).hasClass('press') == true) {
```

Comment:
I think you can remove the == true part, same with the false statement.

$(this).hasClass('press') == false  => (!$(this).hasClass('press'))

Reply:
done